contracts/tonomy/build.sh

```diff
@@ -17,6 +17,9 @@ BUILD_COMMAND="cdt-cpp -abigen -I ${WORKING_DIR}/include -R ${WORKING_DIR}/ricar
 
 echo $BUILD_COMMAND
 
+mkdir -p "${PARENT_PATH}/include/eosio.tonomy"
+cp "${PARENT_PATH}/../eosio.tonomy/include/eosio.tonomy/eosio.tonomy.hpp" "${PARENT_PATH}/include/eosio.tonomy/eosio.tonomy.hpp"
```

Comment:
why u added this?

Reply:
Because this file is now used during compilation. It is a dependency used in native.hpp L12 and native.cpp L6
See https://github.com/Tonomy-Foundation/Tonomy-Contracts/pull/97/files#diff-349ed76354f2750dcd6bba6f40fdc44443e2d2bd3450f070fbac21228808b388R6